packages/blocks/src/page-block/edgeless/components/auto-connect/edgeless-auto-connect-line.ts

```diff
@@ -106,6 +118,9 @@ export class EdgelessAutoConnectLine extends WithDisposable(LitElement) {
           EXPAND_OFFSET / 2,
         ]);
 
+        if (newStart.every(a => isNaN(a)) || newEnd.every(a => isNaN(a)))
```

Comment:
I think we should avoid using `isNaN` that works _after_ the invalid data is generated, but to add testing condition for the root cause and avoid `NaN`. The `isNaN` should not used across the codebase, unless for generic data conversion testing.